paprika/simulate.py

```diff
@@ -0,0 +1,160 @@
+import numpy as np
+import subprocess as sp
+import logging as log
+import os as os
+import parmed as pmd
+from collections import OrderedDict
+
+
+def _write_input_file(filename, input_dict, title='Input.'):
```

Comment:
Not sure about leading underscore function names outside of a class. Probably fine for now, but we may want to deal with this later.

Reply:
I anticipate this function to be used by at least two `simulate` classes, but not something that external code would call since it requires the class provide the data in the expected dictionaries, lists, etc.